api/turing/api/ensembling_job_api.go

```diff
@@ -67,5 +118,22 @@ func (c EnsemblingJobController) Routes() []Route {
 			handler: c.Create,
 			body:    models.EnsemblingJob{},
 		},
+		{
+			method:  http.MethodGet,
+			path:    "/projects/{project_id}/jobs/{id}",
+			handler: c.GetEnsemblingJob,
+		},
+		{
+			method:  http.MethodGet,
+			path:    "/projects/{project_id}/jobs",
+			handler: c.ListEnsemblingJobs,
+		},
 	}
 }
+
+// GetEnsemblingJobOptions is the options that can be parsed
+// from query params for the GET ensembling job method
+type GetEnsemblingJobOptions struct {
```

Comment:
Nitpick: For consistency with the [Ensemblers API](https://github.com/gojek/turing/blob/main/api/turing/api/ensemblers_api.go#L179), can we name this `EnsemblingJobPathOptions`?

Reply:
Ah because in my other PR it's not always just Path options. Sometimes it's used internally by the batch ensembling process. Not all options are used in path. Path is a subset of what list does.